### PR TITLE
Client - Requester - Add task to get help with

### PR DESCRIPTION
**Pull request by pierre-1**
# [PT Story](https://www.pivotaltracker.com/story/show/172275165)
```
As a person in need
To be able to get some help
I would like to be able add a task that someone else can help me with
```
## Changes proposed in this pull request:
* Shows list of products 
* Products are added to a task
* Tasks can now be confirmed
* The user can see a success message once the request has been submitted
* The task containing the products is sent to the back end with a put request

## What we have learned working on this feature:
* Using action types and dispatch 
* Map function over products
* Conditioning a success message on an action
* We cannot start cypress server and tests at the same time in this cypress version but we need to do this with the help of a package

## Packages added
- Axios 
- Redux
- redux-thunk
- start-server-and-test

## Screenshots
Screen Shot 2020-04-15 at 2 11 02 PM
Screen Shot 2020-04-15 at 2 11 07 PM
Screen Shot 2020-04-15 at 2 11 15 PM
Screen Shot 2020-04-15 at 2 11 22 PM

### CODE REVIEW

File: cypress/fixtures/taskList_response.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "task": {
+        "id": 1,
+        "products": [
+            {
+                "amount": 1,
```
Comment (Carltesio) [inline cypress/fixtures/taskList_response.json]: please format this one

File: cypress/fixtures/taskList_response.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+                "name": "Potatoes",
+                "total": "98.0"
+            }
+        ],
```
Comment (Carltesio) [inline cypress/fixtures/taskList_response.json]: please format this one

File: cypress/fixtures/taskList_update_response.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "task": {
+        "id": 1,
+        "products": [
+            {
```
Comment (Carltesio) [inline cypress/fixtures/taskList_update_response.json]: please format this one

File: cypress/integration/userCanSeeTitle.feature.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+describe("Free Article view:", () => {
```
Comment (Carltesio) [inline cypress/integration/userCanSeeTitle.feature.js]: Article?

File: cypress/fixtures/taskList_response.json (hunk 1 above)
Comment (pierre-1) [reply cypress/fixtures/taskList_response.json]: Done

File: cypress/fixtures/taskList_response.json (hunk 2 above)
Comment (pierre-1) [reply cypress/fixtures/taskList_response.json]: Done

File: cypress/fixtures/taskList_update_response.json (hunk 3 above)
Comment (pierre-1) [reply cypress/fixtures/taskList_update_response.json]: Done

File: cypress/integration/userCanSeeTitle.feature.js (hunk 4 above)
Comment (pierre-1) [reply cypress/integration/userCanSeeTitle.feature.js]: Made the change, thanks for catching that!

File: cypress/fixtures/taskList_response.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    "products": [
+      {
+        "amount": 1,
+        "name": "Potatoes",
+        "total": "98.0"
+      }
+    ],
+    "total": "98.0",
+    "order_total": "98.0"
+  },
```
Comment (janko101) [inline cypress/fixtures/taskList_response.json:13]: We're not sending order_total anymore from the BE

**Review by janko101 — APPROVED** [review]

**Pull request merged**